README.md

```diff
@@ -32,15 +32,15 @@ Deploy an embedding model:
 ```bash
 helm install stapi-minilm-l6-v2 substratusai/stapi \
   --set model=all-MiniLM-L6-v2 \
-  --set deploymentAnnotations.lingo-models=text-embedding-ada-002 \
+  --set deploymentAnnotations."lingo\.substratus\.ai/models"=text-embedding-ada-002 \
```

Comment:
do we really need those `\` ? it makes it a bit uggly to read.

Could we do this instead?

```
--set "deploymentAnnotations.lingo.substratus.ai/models=text-embedding-ada-002"
```
I am not sure you even need the double quotes

Reply:
I can't get it to work without them either

Reply:
reading values.yaml from bash heredoc seems best:
```
helm install mistral-7b-instruct substratusai/vllm -f - << EOF
model: mistralai/Mistral-7B-Instruct-v0.1
replicaCount: 0
deploymentAnnotations:
  lingo.substratus.ai/models: mistral-7b-instruct-v0.1
EOF
```